src/viewers/patch-work-plugin/src/index.js

```diff
@@ -1,5 +1,21 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+// import {
+//   Manifest,
+//   Viewport,
+//   SingleTileSource,
+//   CanvasProvider,
+//   OpenSeadragonViewer,
+//   OpenSeadragonViewport,
+//   AnnotationDetail,
+//   AnnotationCanvasRepresentation,
+//   Fullscreen,
+//   FullPageViewport,
+//   Bem,
+//   withBemClass,
+// } from 'canvas-panel-beta/lib/legacy';
+
+// Temp until resolve issue with BemModifiers not reaching pins
```

Comment:
@stephenwf I've left this in here for now, but we can switch over and remove `@canvas-panel` when we get the pins issue resolved

Reply:
You should be able to update to `1.1.1` now. Wrapping the patchwork in:
```
<Bem
  prefix="patchwork-"
  cssClassMap={{
    annotation: 'annotation-pin',
  }}
>
 ... 
</Bem>
```

work for me to get the class names working again. I've split off a patchwork in #31